specs/wp-log-in-out-spec.js

```diff
@@ -136,6 +203,140 @@ test.describe( `[${host}] Authentication: (${screenSize}) @parallel @jetpack`, f
 			} );
 		} );
 	}
+
+	if ( dataHelper.hasAccountWithFeatures( '+passwordless +2fa-sms' ) ) {
+		test.describe( 'Can Log in on a passwordless account', function() {
+			test.describe( 'Can request a magic link email by entering the email of an account which does not have a password defined', function() {
+				let magicLoginLink, loginFlow, magicLinkEmail, emailClient;
+				test.before( function () {
+					loginFlow = new LoginFlow( driver, [ '+passwordless',  '+2fa-sms' ] );
+					emailClient = new EmailClient( get( loginFlow.account, 'mailosaur.inboxId' ) );
+					return loginFlow.login();
+				} );
+
+				test.it( 'Can find the magic link in the email received', function() {
+					return emailClient.pollEmailsByRecipient( loginFlow.account.email ).then( function( emails ) {
+						magicLinkEmail = emails.find( email => email.subject.indexOf( 'WordPress.com' ) > -1 );
+						assert( magicLinkEmail !== undefined, 'Could not find the magic login email' );
+						magicLoginLink = magicLinkEmail.html.links[0].href;
+						assert( magicLoginLink !== undefined, 'Could not locate the magic login link in the email' );
+						return true;
+					} );
+				} );
+
+				test.describe( 'Can use the magic link and the code received via sms to log in', function() {
+					let magicLoginPage, twoFALoginPage, twoFACode;
+					test.before( function( done ) {
+						driver.get( magicLoginLink );
+						magicLoginPage = new MagicLoginPage( driver );
+						// make sure we listen for SMS before we trigger any
```

Comment:
> before we trigger any [what]

Looks like that comment misses something at it's end ?

Reply:
any sms?
I'll try to rephrase it as it doesn't seem very clear right now